Makefile

```diff
@@ -258,10 +259,8 @@ bundle/redhat: bundle
 
 # e2e
 .PHONY: e2e-kind
-e2e-kind:
-ifeq (,$(shell kind get clusters ))
-	$(KIND) --kubeconfig="${KUBECONFIG}" create cluster --image=kindest/node:v$(ENVTEST_K8S_VERSION) --config tests/e2e/kind.yaml
-endif
+e2e-kind: kind
+	hack/kind/start-kind.sh
```

Comment:
Here, the target k8s version is no longer passed, and there's no default in the script itself. Better to fix it + maybe pass the version in start-kind target as well.

Reply:
If `KIND_NODE_VERSION` is unset, kind should use whatever is default in the installed kind version.
https://github.com/grafana/grafana-operator/pull/1945/files#diff-839571e93aae150002c0a08bf465f55287add5a13806b70e59cfeeba53854ae6R17-R20
But pinning in the Makefile makes it clear

Reply:
Please, feel free to make further changes if needed :)

Reply:
Since the if condition testing the presence of `KIND_NODE_VERSION` is always true, I removed it 😄